Failure with php72 + php73 sets on PHP 7.2

| Subject        | Details                                                      |
| :------------- | :----------------------------------------------------------- |
| Rector version | v0.6.1 |
| PHP version    | PHP 7.2.25 |
| Full Command   | vendor/bin/rector process src/BadClass.php --dry-run | 

## Current Behaviour 
The following rector.yaml
```yaml
parameters:
    sets:
        - 'php72'
        - 'php73'
```

causes a crash with this message:
```
[ERROR] Could not process "src/BadClass.php" file by "Rector\Php72\Rector\ConstFetch\BarewordStringRector", due to:    
         "Look at "Rector\Php72\Rector\ConstFetch\BarewordStringRector::refactor()" on line 57".  
```

Debug info:
```
vendor/bin/rector process src/BadClass.php --dry-run --debug
File "/home/marketmaker/rector/vendor/autoload.php" is about to be loaded in "AutoloadIncluder::includeCwdVendorAutoloadIfExists()" on line 65"
File "/home/marketmaker/rector/vendor/rector/rector/../../autoload.php" is about to be loaded in "AutoloadIncluder::autoloadProjectAutoloaderFile()" on line 97"
Rector v0.6.1
Config file: rector.yaml

[parsing] src/BadClass.php
[refactoring] src/BadClass.php
    [applying] Rector\Renaming\Rector\Function_\RenameFunctionRector
    [applying] Rector\Php72\Rector\FuncCall\GetClassOnNullRector
    [applying] Rector\Php72\Rector\FuncCall\IsObjectOnIncompleteClassRector
    [applying] Rector\Php72\Rector\FuncCall\ParseStrWithResultArgumentRector
    [applying] Rector\Php72\Rector\FuncCall\StringsAssertNakedRector
    [applying] Rector\Php72\Rector\FuncCall\CreateFunctionToAnonymousFunctionRector
    [applying] Rector\Php72\Rector\FuncCall\StringifyDefineRector
    [applying] Rector\Php73\Rector\FuncCall\ArrayKeyFirstLastRector
    [applying] Rector\Php73\Rector\FuncCall\SensitiveDefineRector
    [applying] Rector\Php73\Rector\FuncCall\StringifyStrNeedlesRector
    [applying] Rector\Php73\Rector\FuncCall\JsonThrowOnErrorRector
    [applying] Rector\Php73\Rector\FuncCall\RegexDashEscapeRector
    [applying] Rector\Php73\Rector\FuncCall\RemoveMissingCompactVariableRector
    [applying] Rector\Php73\Rector\String_\SensitiveHereNowDocRector
    [applying] Rector\Php72\Rector\ConstFetch\BarewordStringRector
    [applying] Rector\Php73\Rector\ConstFetch\SensitiveConstantNameRector
    [applying] Rector\Php72\Rector\ConstFetch\BarewordStringRector
PHP Fatal error:  Uncaught Rector\Exception\ShouldNotHappenException: Look at "Rector\Php72\Rector\ConstFetch\BarewordStringRector::refactor()" on line 57 in /home/marketmaker/rector/vendor/rector/rector/packages/Php72/src/Rector/ConstFetch/BarewordStringRector.php:57
Stack trace:
#0 /home/marketmaker/rector/vendor/rector/rector/src/Rector/AbstractRector.php(116): Rector\Php72\Rector\ConstFetch\BarewordStringRector->refactor(Object(PhpParser\Node\Expr\ConstFetch))
#1 /home/marketmaker/rector/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php(123): Rector\Rector\AbstractRector->enterNode(Object(PhpParser\Node\Expr\ConstFetch))
#2 /home/marketmaker/rector/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php(223): PhpParser\NodeTraverser->traverseNode(Object(PhpParser\Node\Arg))
#3 /home/marketmaker/rector/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php(114): PhpParser\NodeTraverser->traverseArray(Array)
#4 /home/marketmaker/rector/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php(146): PhpParser\NodeTravers in /home/marketmaker/rector/vendor/rector/rector/packages/Php72/src/Rector/ConstFetch/BarewordStringRector.php on line 57
Fatal error: Uncaught Rector\Exception\ShouldNotHappenException: Look at "Rector\Php72\Rector\ConstFetch\BarewordStringRector::refactor()" on line 57 in /home/marketmaker/rector/vendor/rector/rector/packages/Php72/src/Rector/ConstFetch/BarewordStringRector.php:57
Stack trace:
#0 /home/marketmaker/rector/vendor/rector/rector/src/Rector/AbstractRector.php(116): Rector\Php72\Rector\ConstFetch\BarewordStringRector->refactor(Object(PhpParser\Node\Expr\ConstFetch))
#1 /home/marketmaker/rector/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php(123): Rector\Rector\AbstractRector->enterNode(Object(PhpParser\Node\Expr\ConstFetch))
#2 /home/marketmaker/rector/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php(223): PhpParser\NodeTraverser->traverseNode(Object(PhpParser\Node\Arg))
#3 /home/marketmaker/rector/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php(114): PhpParser\NodeTraverser->traverseArray(Array)
#4 /home/marketmaker/rector/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php(146): PhpParser\NodeTravers in /home/marketmaker/rector/vendor/rector/rector/packages/Php72/src/Rector/ConstFetch/BarewordStringRector.php on line 57
```

<!-- report from command line with --debug -->
<!-- description/screenshot -->

## Minimal PHP Code Causing Issue

<!-- if possible, always run Rector on specfic file, that causes the error -->
<!-- e.g. vendor/bin/rector process /src/SpecificFile.php --set dead-code --debug -->  

```php
<?php

namespace Test;

class BadClass
{
    private function badMethod()
    {
        json_decode('{}');
    }
}

--- a/packages/Php72/src/Rector/ConstFetch/BarewordStringRector.php
+++ b/packages/Php72/src/Rector/ConstFetch/BarewordStringRector.php
@@ -54,7 +54,8 @@ final class BarewordStringRector extends AbstractRector
         // load the file!
         $fileInfo = $node->getAttribute(AttributeKey::FILE_INFO);
         if ($fileInfo === null) {
-            throw new ShouldNotHappenException();
+            // unable to process
+            return null;
         }
 
         $this->undefinedConstants = [];

